MAINTAINERS: clarify abilities of org members (#5123)
tldr-pages/tldr: MAINTAINERS.md

## Organization members

> In addition to everything that a repository collaborator can do, organization members

Author: 😆 
```suggestion
In addition to everything that repository collaborators can do, organization members
```

> have collaborator access to all the repositories in the tldr-pages organization.

Reviewer: The phrasing changes are 👍, but while I agree that "write access" is not clear enough, I'm not sure organization members can be described to simply have collaborator access to all repos in the org. I believe they have [additional permissions](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/permission-levels-for-an-organization), though the exact diff is not exactly clear on my mind.

Perhaps it's not important to be exhaustive here. Maybe we can just add a link here, something like this?

```suggestion
have collaborator access to all the repositories in the tldr-pages organization,
in addition to [a few extra maintenance tools](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/permission-levels-for-an-organization).
```

Author: Okay.  Interesting.

> also has collaborator access to all the repositories in the tldr-pages organization.

Author: 😄 
```suggestion
also have collaborator access to all the repositories in the tldr-pages organization.
```

> also have collaborator access to all the repositories in the tldr-pages organization.

Author: `also` is redundent here.
```suggestion
have collaborator access to all the repositories in the tldr-pages organization.
```

## Organization members

In addition to everything that repository collaborators can do, organization members

> have collaborator access to all the repositories in the tldr-pages organization,

Reviewer: Hmm, re-reading the previous section's description and this one, I don't think it's necessary to swap "write access" to "collaborator access":

- The previous description spells out more specifically what "write access" weans
- This description already mentions "In addition to everything that repository collaborators can do"

So I'd suggest restoring the "write access" phrasing, to avoid unnecessary repetition.

Author: Okay.  I'll do that.  I changed it because it I couldn't tell if it actually meant collaborator access, but then, I didn't read the whole thing like you did.

Author: ```suggestion
have write access to all the repositories in the tldr-pages organization,
```

Author: Those are good points :)

> in addition to [a few extra maintenance tools](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/permission-levels-for-an-organization).

Author: I like this better.
```suggestion
and [a few extra maintenance tools](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/permission-levels-for-an-organization).
```

Reviewer 2: This reads better as it does not repeat the wording.
